common/data_refinery_common/__init__.py

```diff
@@ -0,0 +1,12 @@
+from django.apps import AppConfig
+
+
+class CommonAppConfig(AppConfig):
+    name = "data_refinery_common"
+
+    # This will be called multiple times
+    def ready(self):
+        import data_refinery_common.signals
+
+
+default_app_config = "data_refinery_common.CommonAppConfig"
```

Comment:
What's this file doing?

Reply:
This is modifying the app config so that it imports `signals.py`

Reply:
Oh I see... In resources portal we've just been adding the signals to the model file. If you do that does this become unnecessary? I think I actually prefer having the signals in the model file because it's more obvious why a model would behave differently.